package.json

```diff
@@ -98,7 +98,7 @@
     "build": "yarn run clean && yarn run build-rb && yarn run lint && yarn run dev",
     "build-ci": "./node_modules/.bin/webpack --progress --colors --config build/webpack.config.js",
     "build-rb": "mojito-rb-gen -s src/i18n -o src/i18n/json -b en-US.properties",
-    "ci": "yarn run clean && yarn run build-rb && yarn run lint && yarn run test && yarn run build-ci",
+    "ci": "yarn run clean && yarn run build-rb && yarn run build-ci && yarn run lint && yarn run test && yarn run functional-tests-ci",
```

Comment:
I believe the reason we were timing out was due to us building right before running the tests. This updates the `dist` folder which is being served by the python SimpleHTTPServer. By moving the build to before the unit tests, it gives the simple server plenty of time to update before running our functional tests.